test/test_torch.py

```diff
@@ -15495,8 +15495,34 @@ def test_min(self, device, dtype):
     @dtypesIfCUDA(torch.half, torch.float)
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
     def test_minmax(self, device, dtype):
-        self._test_minmax_helper(lambda x: torch._min_max(x)[0], np.min, device, dtype, skip_indices=True)
-        self._test_minmax_helper(lambda x: torch._min_max(x)[1], np.max, device, dtype, skip_indices=True)
+
+        def _min_wrapper(x, dim=None, keepdims=False):
```

Comment:
You never call this with `dim` that is not None and  keepdims=True?

Reply:
the dims are cycled through in `_test_minmax_helper`.

however, I didn't know about `np.amin/np.amax` when writing this code. After seeing the PR you linked to we can just rename `torch._min_max` to `torch._aminmax` to simplify this further.  If it's ok, let me do that in a separate PR.

Reply:
One would think I should know that since I wrote _test_minmax_helper :-) Yeah, let's do renaming in a separate PR (and then another renaming to `aminmax` without underscore when everything is ready).

Reply:
fixed naming in https://github.com/pytorch/pytorch/pull/44001, further up the stack